mvpa2/generators/permutation.py

```diff
@@ -94,6 +97,10 @@ def _get_pcfg(self, ds):
         return get_limit_filter(self._limit, collection)
 
 
+    def get_chunk_attr(self, ds):
```

Comment:
like what? that function is here because  _permute* functions doesn't have access to ds, so you cannot get chunk attributes directly or call that function within _permute* to return chunk attributes. It deserve a comment anyway